training/mochi-1/prepare_dataset.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+MODEL_ID="genmo/mochi-1-preview"
+
+NUM_GPUS=1
+
+# For more details on the expected data format, please refer to the README.
+DATA_ROOT="/home/sayak/cogvideox-factory/video-dataset-disney"  # This needs to be the path to the base directory where your videos are located.
+CAPTION_COLUMN="prompt.txt"
+VIDEO_COLUMN="videos.txt"
+OUTPUT_DIR="/home/sayak/cogvideox-factory/video-dataset-disney/mochi-1/preprocessed-dataset"
+HEIGHT_BUCKETS="480"
+WIDTH_BUCKETS="848"
+FRAME_BUCKETS="1 84"
```

Comment:
@a-r-r-o-w to account for what I mentioned here:
https://github.com/a-r-r-o-w/cogvideox-factory/pull/90#discussion_r1847602195

Reply:
Would making 1 the default be more sensible, or do you think better for user to configure this? The change I applied in our other PR took care of this in a similar fashion by always defaulting to 1 if a suitable frame bucket could not be found:

![image](https://github.com/user-attachments/assets/0eac6716-4721-43e8-9ea2-ef412152d0a0)

Reply:
The `1, 84` because of what I mentioned here:
https://github.com/a-r-r-o-w/cogvideox-factory/pull/90#discussion_r1847602195